### PR TITLE
Feat: added List

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

**Review by Shah-Arsalan — APPROVED** [review]
Looks good .  I think it can be merged

File: doc_styles/doc_list.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.doc-list-container{
+    margin: 1rem;
+    display: flex;
+    /* flex-direction: column; */
```
Comment (romabulani) [inline doc_styles/doc_list.css]: Remove comment if not in use

File: components/list/list.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+</head>
+<body>
+    <div class="doc-list-container">
+        <ul class="list-spaced">
+            <div class="list-spaced-heading"><h3>List Heading</h3></div>
+            <a href="#">
+                <li class="list-spaced-item text-md font-wt-semibold list-item-active">Active Item</li>
+            </a>
+            <a href="#">
+            <li class="list-spaced-item text-md font-wt-semibold">Second Item</li>
```
Comment (romabulani) [inline components/list/list.html]: if possible, give proper indentation

File: components/list/list.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    cursor: pointer;
+}
+.list-stacked{
+    list-style-type: none;
+    border-left: 1px solid var(--text-color);
+    border-bottom: 1px solid var(--text-color);
+    border-right: 1px solid var(--text-color);
+    display: flex;
+    flex-direction: column;
+    /* width: 40rem; */
```
Comment (romabulani) [inline components/list/list.css]: Remove comment if not in use

**Review by romabulani — APPROVED** [review]
You can merge after these changes

**Review by PeeyushPrashant — APPROVED** [review]
looks good

File: components/list/list.css (hunk 3 above)
Comment (Rohanmond) [reply components/list/list.css]: done . Thanks for the review

**Pull request merged**